udata_front/theme/gouvfr/assets/js/components/MultiSelect/MultiSelect.vue

```diff
@@ -261,6 +267,10 @@ export default defineComponent({
       image: obj.logo_thumbnail ?? obj.logo ?? obj.image_url ?? obj.image,
       hidden: obj.hidden,
       selected: !!obj.selected,
+      ...(props.hasLink && {
```

Comment:
Is this working when props.hasLink is false ? It looks like this will try to spread a boolean value

Reply:
That was the point, if you don't want to show a link, you can set hasLink to false and then prevent potential pages associated with the object to appear as a link

Reply:
Can you add stories with hasLink set to true / false ?